README.md

```diff
@@ -56,24 +71,24 @@ packages.
 
  * Save and exit.
 
- * Run `make` (for a real benchmarking setup do *not* use `-j`)
-
  * Run `make deb-pkg`
 
-## Step 3: Install and Boot the New Kernel
+`-j` should not be used when building the kernel. This helps to reduce
```

Comment:
I don't think we should mention this (we had a discussion about this a few weeks back: if we have to tell people everything they *shouldn't* do, we'll be here all day; we have to assume that people only do what we tell them, and no more).

Reply:
In another PR this will become the "how to make an krun kernel binary for general distribution" section, so it probably is worth mentioning to ensure the distribution gets built properly?

I still think people will be tempted to invoke -j personally, but I know you don't share this opinion.

Reply:
If we haven't told people to use `-j` they shouldn't be doing it: just as they shouldn't do a zillion other things we haven't told them not to do.

Reply:
Sure, but it's a well known standard trick to make things go faster ;)

I don't feel strongly though.